batch/batch/front_end/templates/batches.html

```diff
@@ -1,11 +1,14 @@
 {% extends "layout.html" %}
 {% block title %}Batches{% endblock %}
+{% block head %}
+  <script src="{{ base_path }}/common_static/focus_on_keyup.js"></script>
+{% endblock %}
 {% block content %}
   <h1>Batches</h1>
   <div class="flex-col" style="width: 100%; min-width: 653px; max-width: 1024px;">
     <div class="flex-col-align-right">
 	<form method="GET" action="{{ base_path }}/batches">
-	  <input style="vertical-align:text-bottom;" name="q" size=30 type="text"
+	  <input style="vertical-align:text-bottom;" id="searchBar" name="q" size=30 type="text"
```

Comment:
If we change this name we have to change it pervasively. There's a couple hidden inputs at the bottom of this page that are used for paging the batches. There's references to `q` in `front_end/front_end.py`. It seems easier to not change it.

Reply:
I changed the `id`. I thought that's different than name. https://www.tutorialspoint.com/What-is-the-difference-between-id-and-name-attributes-in-HTML